ivan-a-kovtun/lab1-BasicIO/main.c

```diff
@@ -0,0 +1,31 @@
+/******************************************************************************/
+/*  Files to Include                                                          */
+/******************************************************************************/
+#ifdef __XC32
+    #include <xc.h>          /* Defines special function registers, CP0 regs  */
+#endif
+
+#include <stdint.h>         /* For uint32_t definition                        */
+#include <stdbool.h>        /* For true/false definition                      */
+
+#include "system.h"         /* System funct/params, like osc/periph config    */
+#include "user.h"           /* User funct/params, such as InitApp             */
+
+/******************************************************************************/
+/* Global Variable Declaration                                                */
+/******************************************************************************/
+
+/* i.e. uint32_t <variable_name>; */
+
+/******************************************************************************/
+/* Main Program                                                               */
+/******************************************************************************/
+
+int32_t main(void)
+{
+    /* Initialize I/O 
+     * and Peripherals
+     * for application */
+    InitApp();
```

Comment:
Use consistent naming scheme. Either `init_app' & 'scan_leds', 'InitApp' & 'ScanLEDs', or any other coding style. But it should be consistent